libraries/Microsoft.Bot.Schema/SuggestedActions.cs

```diff
@@ -35,10 +35,10 @@ public SuggestedActions()
         /// shown to.  These Ids are relative to the channelId and a subset of
         /// all recipients of the activity</param>
         /// <param name="actions">Actions that can be shown to the user</param>
-        public SuggestedActions(IList<string> to = default(IList<string>), IList<CardAction> actions = default(IList<CardAction>))
+        public SuggestedActions(IEnumerable<string> to = default(IEnumerable<string>), IEnumerable<CardAction> actions = default(IEnumerable<CardAction>))
         {
-            To = to;
-            Actions = actions;
+            To = to?.ToList();
```

Comment:
I don't understand why all of these null checks have been added. I don't understand why we're allowing Null's here at all, and I would *strongly* vote to remove that code.

Reply:
fixing

Reply:
@cleemullins , Should the SuggestedActions construct throw if **to** is null or empty?